README.md

```diff
@@ -1 +1,47 @@
-# Awesome-Book
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+In this project, we built a basic website that allows users to add/remove books from a list. This was achieve by using JavaScript objects, arrays, modifying the DOM dynamically and adding basic events Listeners.
+
+## Live Demo
+
```

Comment:
Please add instructions to set up the project locally as it is necessary for the non-technical user of your project.